### PR TITLE
adding badge component

### PR DESCRIPTION
**Pull request by bipin7yadav**

### CODE REVIEW

File: badge.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+/* #span2{
+    position: absolute;
+    margin-top: -2rem;
+    margin-left: -1.2rem;
+    background-color: blueviolet;
+    border-radius:50% ;
+    font-size: 2rem;
+    padding: .2rem 1rem;
+} */
```
Comment (manojkumar8400) [inline badge.css:120]: suggestion -: Remove unused comments.

**Review by manojkumar8400 — APPROVED** [review]

File: badge.css [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    gap: 5px;
+    position: relative;
+    /* z-index: 1; */
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 5px;
+    background-color: rgb(155, 173, 238);
```
Comment (mimansha-swarup) [inline badge.css:21]: use variable

File: badge.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    display: flex;
+    gap: 5px;
+    position: relative;
+    /* z-index: 1; */
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 5px;
```
Comment (mimansha-swarup) [inline badge.css:20]: use consistent unit somewhere u are using px somewhere rem
i will suggest to stick with rem

File: badge.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    width: 100%;
+}
+.container{
+    /* background-color:skyblue ; */
+    left: 0;
+    padding: 3px;
+    display: flex;
+    gap: 5px;
+    position: relative;
+    /* z-index: 1; */
```
Comment (mimansha-swarup) [inline badge.css:14]: remove unnecessary comments

File: badge.css [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+}
+
+.off{
+    position: absolute;
+    background-color: red;
+    padding: 2px;
+    border-radius: 50%;
+    bottom: 0;
+    margin-left: -3rem;
```
Comment (mimansha-swarup) [inline badge.css:82]: i guess margin-left is not necessary you can use left instead depends on your use-case too

**Review by mimansha-swarup — CHANGES_REQUESTED** [review]
remove unnecessary comments
use variable for colors
use consistent unit

**Pull request merged**